CONTRIBUTING.md

```diff
@@ -29,6 +29,9 @@ pip install --upgrade pip
 
 # Install dependencies
 pip install -r requirements.txt
+
+# Code formatting (Run this as a pre-commit step for your code changes)
+black .
```

Comment:
we should add this as a part of the validation presubmit job

Reply:
pursuing adding this test job here: https://github.com/openshift/release/pull/45980

Reply:
and as a "make verify" target in this repo.

Reply:
Acknowledged. Adding it as `make verify` target in this repo.